test/utils/apiValidators.test.ts

```diff
@@ -0,0 +1,139 @@
+import * as validation from 'utils/validation'
+import {
+  apiObjectIdValidation,
+  apiAttributeValidation,
+  BAD_REQUEST_BODY_PREFIX,
+  apiCategoryValidation,
+  apiItemDefinitionValidation,
+  apiUserValidation,
+} from 'utils/apiValidators'
+import * as validators from 'utils/validators'
+import { ApiError } from 'utils/types'
+
+describe('apiValidators', () => {
+  const validationFailedReturn = {
+    success: false,
+    message: 'Validation Failed',
+  }
+
+  const validationSuccessReturn = {
+    success: true,
+    message: '',
+  }
+
+  describe('apiValidateObjectId', () => {
+    test('invalid object id throws api error', () => {
+      jest.spyOn(validation, 'validateObjectId').mockReturnValue(false)
+      try {
+        apiObjectIdValidation('3')
```

Comment:
there's also the possibility of a 24(?)-character string that is also an invalid objectID. Might want to make a test for that?